Functions/QualityControl/xASL_qc_GenerateReport.m

```diff
@@ -0,0 +1,418 @@
+function [x,config] = xASL_qc_GenerateReport(x, subject)
+% xASL_qc_GenerateReport Print output QC
+%
+% FORMAT: xASL_qc_GenerateReport(x)
+%
+% INPUT:
+%   x           - structure containing fields with all information required to run this submodule (REQUIRED)
+%   subject     - subject name (OPTIONAL, default = x.SUBJECT)
+%
+% OUTPUT: n/a
+% OUTPUTFILE:
+%   xASL_Report_SubjectName.pdf - printed PDF rapport containing QC images & values
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION: This function iterates over all values in x.Output and all
+%              images in x.Output_im, and prints them in a PDF file.
+%              x.Output & x.Output_im should contain the QC/result output
+%              of all ExploreASL pipeline steps.
+% 
+% EXAMPLE: xASL_qc_GenerateReport(x);
+% __________________________________
+% Copyright (C) 2015-2023 ExploreASL
+
+if ~usejava('jvm') % only if JVM loaded
+    fprintf('Warning: skipping PDF report, JVM missing\n');
+    config = NaN;
+    return;
+end
+
```

Comment:
please add check for the first input parameter

Reply:
Resolved and Added